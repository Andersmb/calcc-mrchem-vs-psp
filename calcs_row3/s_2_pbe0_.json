{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.0772282795474304
          ],
          "radius": 1.09
        },
        {
          "center": [
            0.0,
            0.0,
            2.4568942714525694
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "s",
          "xyz": [
            0.0,
            0.0,
            -0.0772282795474304
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            2.4568942714525694
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "s_2_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -1.0223196500547817e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.30823826743766,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -6.217711020985567e-06,
            1.0135397802172085e-06,
            0.3082382673732825
          ],
          "vector_el": [
            -6.217805783775843e-06,
            1.0134451159382607e-06,
            -0.9130035312441972
          ],
          "vector_nuc": [
            9.476279027541376e-11,
            9.466427894769464e-11,
            1.2212417986174797
          ]
        }
      },
      "geometry": [
        {
          "symbol": "S",
          "xyz": [
            0.0,
            0.0,
            -0.0772282795474304
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            2.4568942714525694
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -89.05670246165987,
          -8.035580753327963,
          -5.98023841941249,
          -5.96843561003755,
          -5.961366284379798,
          -0.760162860256558,
          -0.4270740951851966,
          -0.340448651676319,
          -0.28927454945918335,
          -89.05085955931902,
          -8.029788999606588,
          -5.965115451360108,
          -5.959688833579242,
          -5.958102239168802,
          -0.714006132179768,
          -0.41294909425759424,
          -0.2751407197248039
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -233.18493471459087
      },
      "scf_energy": {
        "E_ee": 183.80843255985116,
        "E_eext": 0.0,
        "E_el": -404.9401565569749,
        "E_en": -960.9680300961353,
        "E_kin": 398.0896446352889,
        "E_next": 0.0,
        "E_nn": 6.313822507789206,
        "E_nuc": 6.313822507789206,
        "E_tot": -398.6263340491857,
        "E_x": -6.336750963305428,
        "E_xc": -19.53345269267427,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 184.01876710701788,
        "E_eext": 0.0,
        "E_el": -402.9272168002871,
        "E_en": -956.7543254599912,
        "E_kin": 395.6488147528826,
        "E_next": 0.0,
        "E_nn": 6.313822507789206,
        "E_nuc": 6.313822507789206,
        "E_tot": -396.6133942924979,
        "E_x": -6.332713788311194,
        "E_xc": -19.50775941188523,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 183.29490720129354,
              "E_eext": 0.0,
              "E_el": -404.87095304006726,
              "E_en": -956.8377312766894,
              "E_kin": 394.46228844164256,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.55713053227805,
              "E_x": -6.319204278373929,
              "E_xc": -19.471213127939947,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.55713053227805,
            "energy_update": 1.9437362397801508,
            "mo_residual": 0.1630835256939863,
            "wall_time": 82.293390144
          },
          {
            "energy_terms": {
              "E_ee": 183.62628121623982,
              "E_eext": 0.0,
              "E_el": -404.9285314420959,
              "E_en": -959.1712968889925,
              "E_kin": 396.45107000097136,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6147089343067,
              "E_x": -6.328779610945785,
              "E_xc": -19.505806159368714,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6147089343067,
            "energy_update": 0.057578402028639175,
            "mo_residual": 0.05202602498090549,
            "wall_time": 61.861013733
          },
          {
            "energy_terms": {
              "E_ee": 183.67404329701378,
              "E_eext": 0.0,
              "E_el": -404.93689600329833,
              "E_en": -960.090460465021,
              "E_kin": 397.32855903166114,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6230734955091,
              "E_x": -6.3317747387148025,
              "E_xc": -19.517263128237477,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6230734955091,
            "energy_update": 0.008364561202427012,
            "mo_residual": 0.022829209549227743,
            "wall_time": 52.282972691
          },
          {
            "energy_terms": {
              "E_ee": 183.7884842655537,
              "E_eext": 0.0,
              "E_el": -404.93930859037454,
              "E_en": -960.8234570830739,
              "E_kin": 397.96142009338917,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6254860825853,
              "E_x": -6.335579593423217,
              "E_xc": -19.530176272820327,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6254860825853,
            "energy_update": 0.0024125870762077284,
            "mo_residual": 0.01973452462122158,
            "wall_time": 48.031654471
          },
          {
            "energy_terms": {
              "E_ee": 183.82334321821926,
              "E_eext": 0.0,
              "E_el": -404.93992044019575,
              "E_en": -961.1624453486354,
              "E_kin": 398.27206100188135,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.62609793240654,
              "E_x": -6.337211140493807,
              "E_xc": -19.535668171167245,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.62609793240654,
            "energy_update": 0.0006118498212117629,
            "mo_residual": 0.014048102291050054,
            "wall_time": 49.160791949
          },
          {
            "energy_terms": {
              "E_ee": 183.81735776736954,
              "E_eext": 0.0,
              "E_el": -404.9400757623807,
              "E_en": -961.0357886074639,
              "E_kin": 398.14976515362827,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.62625325459146,
              "E_x": -6.336953851479176,
              "E_xc": -19.534456224435463,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.62625325459146,
            "energy_update": 0.00015532218492353422,
            "mo_residual": 0.008416087430296204,
            "wall_time": 49.47187246
          },
          {
            "energy_terms": {
              "E_ee": 183.81478728033392,
              "E_eext": 0.0,
              "E_el": -404.9401181601033,
              "E_en": -960.9494543858656,
              "E_kin": 398.0650669156165,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.62629565231407,
              "E_x": -6.336848039727119,
              "E_xc": -19.533669930461034,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.62629565231407,
            "energy_update": 4.239772260916652e-05,
            "mo_residual": 0.005082705953915271,
            "wall_time": 50.484374134
          },
          {
            "energy_terms": {
              "E_ee": 183.8054331465186,
              "E_eext": 0.0,
              "E_el": -404.94013420776304,
              "E_en": -960.9371109532601,
              "E_kin": 398.0611952739218,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263116999738,
              "E_x": -6.336641925631145,
              "E_xc": -19.533009749312164,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263116999738,
            "energy_update": 1.604765975571354e-05,
            "mo_residual": 0.004152304158762029,
            "wall_time": 50.93040695
          },
          {
            "energy_terms": {
              "E_ee": 183.80071428667787,
              "E_eext": 0.0,
              "E_el": -404.94014354842784,
              "E_en": -960.930673324128,
              "E_kin": 398.0590913402875,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.62632104063863,
              "E_x": -6.3365563671181855,
              "E_xc": -19.53271948414711,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.62632104063863,
            "energy_update": 9.340664803403342e-06,
            "mo_residual": 0.003372909997756495,
            "wall_time": 51.306572316
          },
          {
            "energy_terms": {
              "E_ee": 183.80497789635143,
              "E_eext": 0.0,
              "E_el": -404.94015265562376,
              "E_en": -960.9632058314904,
              "E_kin": 398.0881207637488,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.62633014783455,
              "E_x": -6.336739851558968,
              "E_xc": -19.533305632674686,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.62633014783455,
            "energy_update": 9.107195921842504e-06,
            "mo_residual": 0.0031886999423134505,
            "wall_time": 53.692272203
          },
          {
            "energy_terms": {
              "E_ee": 183.80837000460411,
              "E_eext": 0.0,
              "E_el": -404.9401556278459,
              "E_en": -960.9805525093517,
              "E_kin": 398.1024483273901,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263331200567,
              "E_x": -6.336809539161599,
              "E_xc": -19.53361191132681,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263331200567,
            "energy_update": 2.9722221484007605e-06,
            "mo_residual": 0.001492974541436576,
            "wall_time": 53.988325306
          },
          {
            "energy_terms": {
              "E_ee": 183.8090080327396,
              "E_eext": 0.0,
              "E_el": -404.94015627255413,
              "E_en": -960.9722457370884,
              "E_kin": 398.09339976984097,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263337647649,
              "E_x": -6.336778929714752,
              "E_xc": -19.53353940833155,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263337647649,
            "energy_update": 6.447082228078216e-07,
            "mo_residual": 0.0005354363821633316,
            "wall_time": 54.105262848
          },
          {
            "energy_terms": {
              "E_ee": 183.80892484730992,
              "E_eext": 0.0,
              "E_el": -404.94015639988913,
              "E_en": -960.9662467933205,
              "E_kin": 398.08739607193496,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263338920999,
              "E_x": -6.336757376743169,
              "E_xc": -19.53347314907033,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263338920999,
            "energy_update": 1.273349994335149e-07,
            "mo_residual": 0.00036457391323069927,
            "wall_time": 54.595997618
          },
          {
            "energy_terms": {
              "E_ee": 183.8085849872379,
              "E_eext": 0.0,
              "E_el": -404.9401565099268,
              "E_en": -960.965541813204,
              "E_kin": 398.0869825279627,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263340021376,
              "E_x": -6.336744405290846,
              "E_xc": -19.533437806632453,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263340021376,
            "energy_update": 1.1003766076100874e-07,
            "mo_residual": 0.00019179656512483315,
            "wall_time": 55.598886575
          },
          {
            "energy_terms": {
              "E_ee": 183.80835210396023,
              "E_eext": 0.0,
              "E_el": -404.9401565517755,
              "E_en": -960.9669177760044,
              "E_kin": 398.08859257896745,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263340439863,
              "E_x": -6.3367458559947245,
              "E_xc": -19.53343760270409,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263340439863,
            "energy_update": 4.184869339951547e-08,
            "mo_residual": 0.00013581434194709284,
            "wall_time": 54.456863151
          },
          {
            "energy_terms": {
              "E_ee": 183.80843255985116,
              "E_eext": 0.0,
              "E_el": -404.9401565569749,
              "E_en": -960.9680300961353,
              "E_kin": 398.0896446352889,
              "E_next": 0.0,
              "E_nn": 6.313822507789206,
              "E_nuc": 6.313822507789206,
              "E_tot": -398.6263340491857,
              "E_x": -6.336750963305428,
              "E_xc": -19.53345269267427,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -398.6263340491857,
            "energy_update": 5.199410679779248e-09,
            "mo_residual": 5.475547337972481e-05,
            "wall_time": 62.518311788
          }
        ],
        "wall_time": 884.820973682
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
