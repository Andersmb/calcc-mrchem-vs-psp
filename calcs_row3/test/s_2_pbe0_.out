                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000       -0.077228
    1       H            :        0.000000        0.000000        2.456894
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               9
 Beta electrons          :                                               8
 Total electrons         :                                              17
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999944119e-01
    1    1    a          :                              9.999999447052e-01
    2    1    a          :                              1.000000063453e+00
    3    1    a          :                              9.999999999999e-01
    4    1    a          :                              1.000000002779e+00
    5    1    a          :                              9.999994934541e-01
    6    1    a          :                              1.000000899028e+00
    7    1    a          :                              9.999999999999e-01
    8    1    a          :                              1.000000001866e+00
    9    1    b          :                              9.999999944119e-01
   10    1    b          :                              9.999999447052e-01
   11    1    b          :                              1.000000063453e+00
   12    1    b          :                              9.999999999999e-01
   13    1    b          :                              1.000000002779e+00
   14    1    b          :                              9.999994934541e-01
   15    1    b          :                              1.000000899028e+00
   16    1    b          :                              9.999999999999e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                395.648814752883
 E-N energy              :            (au)               -956.754325459991
 Coulomb energy          :            (au)                184.018767107018
 Exchange energy         :            (au)                 -6.332713788311
 X-C energy              :            (au)                -19.507759411885
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  6.313822507789
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -402.927216800287
 Nuclear energy          :            (au)                  6.313822507789
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.966133942925e+02
                         :      (kcal/mol)             -2.488786604508e+05
                         :        (kJ/mol)             -1.041308316002e+06
                         :            (eV)             -1.079239978808e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.388786513245
    1    1    a          :            (au)                 -7.934509732801
    2    1    a          :            (au)                 -5.885071333188
    3    1    a          :            (au)                 -5.873117467667
    4    1    a          :            (au)                 -5.862596092652
    5    1    a          :            (au)                 -0.755596532212
    6    1    a          :            (au)                 -0.408268260991
    7    1    a          :            (au)                 -0.316771093084
    8    1    a          :            (au)                 -0.267165444429
    9    1    b          :            (au)                -88.383419378048
   10    1    b          :            (au)                 -7.928867928213
   11    1    b          :            (au)                 -5.870224655540
   12    1    b          :            (au)                 -5.865242826036
   13    1    b          :            (au)                 -5.859654418541
   14    1    b          :            (au)                 -0.714410102457
   15    1    b          :            (au)                 -0.387655397975
   16    1    b          :            (au)                 -0.250423387483
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -230.951780564563
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.123106e+00        -396.613394292498         -3.966134e+02
    1          1.630835e-01        -398.557130532278         -1.943736e+00
    2          5.202602e-02        -398.614708934307         -5.757840e-02
    3          2.282921e-02        -398.623073495509         -8.364561e-03
    4          1.973452e-02        -398.625486082585         -2.412587e-03
    5          1.404810e-02        -398.626097932407         -6.118498e-04
    6          8.416087e-03        -398.626253254591         -1.553222e-04
    7          5.082706e-03        -398.626295652314         -4.239772e-05
    8          4.152304e-03        -398.626311699974         -1.604766e-05
    9          3.372910e-03        -398.626321040639         -9.340665e-06
   10          3.188700e-03        -398.626330147835         -9.107196e-06
   11          1.492975e-03        -398.626333120057         -2.972222e-06
   12          5.354364e-04        -398.626333764765         -6.447082e-07
   13          3.645739e-04        -398.626333892100         -1.273350e-07
   14          1.917966e-04        -398.626334002138         -1.100377e-07
   15          1.358143e-04        -398.626334043986         -4.184869e-08
   16          5.475547e-05        -398.626334049186         -5.199411e-09
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000       -0.077228
    1       H            :        0.000000        0.000000        2.456894
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                398.089644635289
 E-N energy              :            (au)               -960.968030096135
 Coulomb energy          :            (au)                183.808432559851
 Exchange energy         :            (au)                 -6.336750963305
 X-C energy              :            (au)                -19.533452692674
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  6.313822507789
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -404.940156556975
 Nuclear energy          :            (au)                  6.313822507789
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.986263340492e+02
                         :      (kcal/mol)             -2.501417992086e+05
                         :        (kJ/mol)             -1.046593288568e+06
                         :            (eV)             -1.084717466688e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.056702461660
    1    1    a          :            (au)                 -8.035580753328
    2    1    a          :            (au)                 -5.980238419412
    3    1    a          :            (au)                 -5.968435610038
    4    1    a          :            (au)                 -5.961366284380
    5    1    a          :            (au)                 -0.760162860257
    6    1    a          :            (au)                 -0.427074095185
    7    1    a          :            (au)                 -0.340448651676
    8    1    a          :            (au)                 -0.289274549459
    9    1    b          :            (au)                -89.050859559319
   10    1    b          :            (au)                 -8.029788999607
   11    1    b          :            (au)                 -5.965115451360
   12    1    b          :            (au)                 -5.959688833579
   13    1    b          :            (au)                 -5.958102239169
   14    1    b          :            (au)                 -0.714006132180
   15    1    b          :            (au)                 -0.412949094258
   16    1    b          :            (au)                 -0.275140719725
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -233.184934714591
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000006        0.000001       -0.913004
 Magnitude               :            (au)                        0.913004
                         :         (Debye)                        2.320623
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.221242
 Magnitude               :            (au)                        1.221242
                         :         (Debye)                        3.104087
---------------------------------------------------------------------------
 Total vector            :       -0.000006        0.000001        0.308238
 Magnitude               :            (au)                        0.308238
                         :         (Debye)                        0.783463
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 18m  7s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
